DigitalLearningSolutions.Web/ViewModels/ForgotPassword/ForgotPasswordViewModel.cs

```diff
@@ -3,5 +3,18 @@
     public class ForgotPasswordViewModel
     {
         public string? EmailAddress;
+
+        public bool ErrorHasOccurred;
+
+        public string? EmailErrorMessage;
```

Comment:
Is ErrorHasOccurred field necessary? Can it be that error occurs and we don't have an error message or vice versa?